plugins/flowdock/samson_flowdock.gemspec

```diff
@@ -1,6 +1,7 @@
-Gem::Specification.new "samson_flowdock", "0.0.0" do |s|
-  s.summary = "Samson flowdock integration"
-  s.authors = ["Michael Grosser"]
-  s.email = "michael@grosser.it"
-  s.add_runtime_dependency "flowdock", "~> 0.3.1"
+Gem::Specification.new 'samson_flowdock', '0.0.0' do |s|
+  s.summary = 'Samson flowdock integration'
+  s.authors = ['Michael Grosser']
+  s.email = 'michael@grosser.it'
```

Comment:
I guess we can also remove personal info from here ... this was just me copy-pasting a local gemspec I had ...